src/cli/flags/repo.go

```diff
@@ -76,12 +76,16 @@ func AddCorsoPassphaseFlags(cmd *cobra.Command) {
 }
 
 // M365 flags
-func AddCorsoUpdatePassphraseFlags(cmd *cobra.Command) {
+func AddUpdatePassphraseFlags(cmd *cobra.Command, require bool) {
 	fs := cmd.Flags()
 	fs.StringVar(&NewPhasephraseFV,
 		NewPassphraseFN,
 		"",
 		"update Corso passphrase for repo")
+
+	if require {
+		cobra.CheckErr(cmd.MarkFlagRequired(NewPassphraseFN))
+	}
```

Comment:
I guess we can always mark it as required. I don't see why we need a flag.

Reply:
Just to make it more flexible if its required in future.